ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/event_handlers.schema.yml

```diff
@@ -0,0 +1,14 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  event_handlers:
+    $ref: "eos_cli_config_gen#/keys/event_handlers"
+    type: list
+    documentation_options:
+      filename: Input Variables
+    description: |
+      Gives the ability to monitor and react to Syslog messages.
+      Event Handlers provide a powerful and flexible tool that can be used to apply self-healing actions,
+      customize the system behavior, and implement workarounds to problems discovered in the field.
```

Comment:
Could this go directly in `eos_cli_config_gen` and be retrived with the `$refs` ?

Reply:
done